app/Http/Controllers/Api/V0/GetLatestController.php

```diff
@@ -33,6 +33,8 @@ public function __invoke(): JsonResponse
                 'ping' => $latest->ping,
                 'download' => ! blank($latest->download) ? Number::bitsToMagnitude(bits: $latest->download_bits, precision: 2, magnitude: 'mbit') : null,
                 'upload' => ! blank($latest->upload) ? Number::bitsToMagnitude(bits: $latest->upload_bits, precision: 2, magnitude: 'mbit') : null,
+                'downloaded_bytes' => ! blank($latest->downloaded_bytes) ? Number::bitsToMagnitude(bits: $latest->downloaded_bytes, precision: 2, magnitude: 'mbit') : null,
```

Comment:
`v0` only exists for backwards compatibility, so no need to add the `_bytes` columns here.